scripts/hud.gd

```diff
@@ -1,12 +1,13 @@
 @tool
+class_name HUD
 extends CanvasLayer
 
 ## The color of the scores and middle line.
 @export var color: Color = Color.WHITE:
 	set = _set_color
 
 ## The size of the score numbers.
-@export_range(50, 500, 0.1) var font_size: float = 200.0:
+@export_range(50, 500, 1) var font_size: float = 200.0:
```

Comment:
I remember that in previous Godot version, if 0.1 wasn't passed, the UI wouldn't show a slider widget.